src/views/quests/QuestView/QuestView.vue

```diff
@@ -89,13 +86,15 @@
   })
   export default class QuestView extends Mixins(PageDataMixin(fetchPageData)) {
     get quest() {
-      return this.pageData.achievement_roadmap.find(p => p.title === this.$route.params.id);
+      return this.pageData.achievement_roadmap.find(
+        (p: AchievementItem) => p.title === this.$route.params.id,
```

Comment:
I think I'll ask @luxaritas to merge my PR into dev, as I doubt I'll finish the rest of the types soon, so this will become unneeded.